### ISSUE TITLE
Ctlra integration for mapping/using hardware controllers?

### ISSUE THREAD

**Issue by mxmilkiib** [ossia/score#1131]
http://openavproductions.com/ctlra
https://github.com/openAVproductions/openAV-Ctlra
@harryhaaren

**Comment by thibaudk** [comment]
:+1:

**Comment by jcelerier** [comment]
yep, the big question would be : what kind of mapping from the controllers would make sense ?

* I guess there are some obvious ones with the transport, etc. 
* Currently selected process could be mapped to faders / pots ? 
* what else ?

**Comment by jcelerier** [comment]
also, does the lib work on mac / windows ?
